docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,164 @@
+[Visual Studio Code](../../computing/webinterface/vscode.md),
+[Jupyter](../../computing/webinterface/jupyter.md) and
+[RStudio](../../computing/webinterface/rstudio.md), as well as an
+[accelerated visualization app](../../computing/webinterface/accelerated-visualization.md)
+for GPU-accelerated visualization and rendering. For demanding computation,
+like running full-scale simulations or training neural networks, you should use
+the command-line interface, since it allows you to access more resources and
+schedule your jobs.
+
+### Command-line interface
+
+While many of the interactive applications in the web interface, such as
+Jupyter and RStudio, are easy to use and thus a good starting point for using
+CSC supercomputers, their computing capacity is limited to relatively
+low-resource interactive usage. If you need access to more resources (e.g.
+multiple CPU nodes or GPUs) or if you require efficiency over interactivity,
+it is a good idea to switch to using the text-based command-line interface to
+interact directly with the supercomputer's
+[Linux operating system](./env-guide/index.md). While this way of working may
+seem archaic, it is truly powerful once you get used to it.
+
+The main benefit of using the CLI is getting to
+[submit your computations as batch jobs](../../computing/running/getting-started.md)
+to the SLURM job scheduler. Requesting a large amount of resources
+for an interactive application like Jupyter typically entails having to wait for
+several hours before finally logging in to manually run the job. This is
+both inconvenient and wasteful, as your interactive session may start at a time
+when you are unable to access your workstation, which means that several hours
+of potential runtime can go unused. When you submit a batch job to SLURM, your
+job is run automatically when the requested resources become available.
+
+You can access the command-line interface either by
+using the [shell applications](../../computing/webinterface/shell.md)
+featured in the web interface (Windows, Linux or macOS) or by
+[using an SSH client on your own workstation](../../computing/connecting.md)
+(Linux or macOS).
+
+## Available resources
+
+### Scientific software
+
+Even if your computations are not highly parallelizable, you can benefit from
+running them on a CSC supercomputer by making use of the variety of useful
+[scientific computing software](../../apps/index.md) that is available.
+Puhti is especially meritorious in this regard, having over a hundred
+pre-installed programs.
+
+CSC systems use [environment modules](../../computing/modules.md) for
+managing software environments. These modules cover everything from
+[compilers](../../computing/installing.md#compiling) to
+[programming languages](../../apps/by_discipline.md#mathematics-and-statistics)
+to workflow utilities like
+[Nextflow](../../apps/nextflow.md) and [Snakemake](../../apps/snakemake.md).
+Running most of our software requires using the
```

Comment:
```suggestion
Running most of the software installed on the supercomputers efficiently requires using the
```
It's not quite "ours", unsure about the efficiently (it could help the point of using CLI over interactive? )...